worker-queue-rabbit/src/main/java/com/github/workerframework/queues/rabbit/WorkerQueueConsumerImpl.java

```diff
@@ -166,10 +166,11 @@ public void processDelivery(Delivery delivery)
                 publishHeaders.put(RABBIT_HEADER_CAF_WORKER_INVALID, ex);
             }
 
-            publisherEventQueue.add(new WorkerPublishQueueEvent(deliveryMessageData, invalidRoutingKey, taskInformation, deliveryHeaders));
+            publisherEventQueue.add(new WorkerPublishQueueEvent(deliveryMessageData, invalidRoutingKey, taskInformation, publishHeaders));
         } catch (final TransientDeliveryException e) {
             LOG.warn("Transient error processing message id {}, disconnecting.", inboundMessageId, e);
             offloadedPayloadsToDelete.remove(inboundMessageId);
+            //Disconnect the channel to allow for a reconnect when the HealthCheck passes.
             disconnectCallback.run();
```

Comment:
Only vaguely related but I see that the readme states  " _- If a transient error occurs in processing, throw a TaskRejectedException_" does that need updated?